### PR TITLE
Module 1: Create navigation bar & header

### PR DESCRIPTION
**Pull request by PolinaStamenova**
**HTML**
- Create nav element
- Create header element

**CSS**
- Styling (setup colors and font related properties)
- Positioning with flexbox
- Use images as a background
 - Create for mobile version

**Linters**
- HTML (linters.yml)
- CSS (.stylelintrc.json)

**README -file**
- Add live demo

_Thank you for your time!_ 🌵 😃

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
       <header>
```
Comment (mricanho) [inline index.html:22]: - [x] Your logo must have a link (a tag)

Comment (mricanho) [inline index.html:22]: - [x] The container of the logo and hamburger must be the header, not a nav tag
- [x] And the hamburger button must be wrapped for a `nav` tag, check [this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)

File: index.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
...
         <h1 class="header-title">I’m Polina Glad to see you!</h1>
         <p>
-          I’m a software developer! I can help you build a product , feature or
+          I’m a software developer! I can help you build a product, feature or
           website Look through some of my work and experience! If you like what
```
Comment (mricanho) [inline index.html]: - [x] The container of the section must be a `section` tag not a `header` tag

File: index.html [hunk 3]
```diff
@@ -26,7 +26,7 @@
...
           you see and have a project you need coded, don’t hestiate to contact
           me.
```
Comment (mricanho) [inline index.html:34]: - [x] Each icon must have a link (`a` tag)
- [x] The button (Let's connect), should be a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design)

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @PolinaStamenova, 

Good job so far! 🔥🔥 Your README file looks professional and the PR title and message are excellent, good commit messages and linters config.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

shine team


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (PolinaStamenova) [reply index.html:32]: On the example the let's connect refers to the icons underneath as it wouldn't make sense to put an anchor tag to the let's connect

**Comment by PolinaStamenova** [comment]
# Changes

- Add hamburger and logo in header tag.
- Add hamburger to nav tag.
- Add link tags to all icons.
- Add link to Let's connect.
- Add section to section tag.
- Add link to Logo.

_Thanks for taking the time to review my PR!_ 😄

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @PolinaStamenova! 🔥 

There you go Polina, you worked hard in all the instructions I gave you👍🏽 

that's how you do it

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**